bot/cogs/fun.py

```diff
@@ -93,34 +93,39 @@ async def eight_ball(self, ctx, *, question=None):
         description="Enter an option between rock, paper, scissor after .play",
         aliases= ["play", "Play", "RPS"],
     )
-    async def rps(ctx, *, response=None):
-        if isinstance(response, str):
-            response = response.lower()
+    async def rps(self, ctx, *, response=None):
+        """Returns strings based on winning/losing/tie"""
+        response = response.lower()
         options = ['rock', 'paper', 'scissors']
         bot_choice = random.choice(options)
-        if response == bot_choice:
+        win = ctx.send("I win... I hope you arent angry?.😂")
+        lose = ctx.send("I lose.. 😶")
+        choose = ctx.send(f'I choose {bot_choice}')
```

Comment:
You are calling `ctx.send` 3 times in a blocking manner, if I am not mistaken, and assigning the return value of those calls to variables.